components/blocks/button.js

```diff
@@ -2,12 +2,20 @@ import Link from "next/link";
 
 import styles from "./button.module.css";
 
-const Button = ({ children, link }) => {
-  return (
-    <Link href={link}>
-      <button className={styles.Button}>{children}</button>
-    </Link>
-  );
+const Button = ({ children, link, target }) => {
+  if (target) {
+    return (
+      <a href={link} target={target}>
```

Comment:
Unrelated to this PR, so we can tackle it separately, but `<a>` tags should be used for links, and `<button>` tags for functionality, so using both here wouldn't be appropriate from an accessibility standpoint. I wonder if we can simplify this code like so:

```
if (target) {
    return (
      <a href={link} target={target} className={styles.Button}>
        {children}
      </a>
    );
  } else {
    return (
      <Link href={link}>
        <a className={styles.Button}>{children}</a>
      </Link>
    );
  }
```

Reply:
Really good point! Learned something new, and fixed on 2ed3f41 🙂